src/icons/general/__tests__/IconSecurityClassification-test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import IconSecurityClassification from '../IconSecurityClassification';
+
+describe('icons/general/IconSecurityClassification', () => {
+    const getWrapper = (props = {}) => shallow(<IconSecurityClassification {...props} />);
+
+    test('should correctly render icon with default values', () => {
+        const wrapper = getWrapper();
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should correctly render icon with all props specified', () => {
+        const wrapper = getWrapper({
+            className: 'test',
+            color: '#987654',
+            height: 100,
+            title: 'title',
+            width: 200,
```

Comment:
`strokeWidth` should also be set, since this test checks for all props on the component.